### PR TITLE
Added ways to detect the malware

### PR DESCRIPTION
**Pull request by AnythingMayBe**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -71,10 +71,14 @@ Before downloading, the malware will create the enclosing directory if it does n
 
 If you find these files, you should delete them immediately, but consider *all* JAR files on your system compromised, and potentially all logins on your web browser as well. Passwords should be changed.
 
+You should also set up your firewall to block all connections to IPs 107.189.3.101 & 85.217.144.130 and to the website https://files-8ie.pages.dev/ip. Please note that if you are suffering a DDoS attack by this malware, you can block the User-Agent "Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:109.0) Gecko/20100101 Firefox/112.0"
```
Comment (Janmm14) [inline README.md:74]: Given they uploaded skyrage payload to this, blocking of ips `95.214.27.172` and `171.22.30.117` is also highly adviced as well as blocking access to `connect.skyrage.de`, `t23e7v6uz8idz87ehugwq.skyrage.de` (`files.skyrage.de`, `file.skyrage.de`, `qw3e1ee12e9hzheu9h1912hew1sh12uw9.skyrage.de`)

Comment (AnythingMayBe) [reply README.md:74]: I wasn't aware of it, that would probably be useful

[the remark this answers is not in the document]
Comment (AnythingMayBe) [reply README.md:74]: That's still IOCs, you can detect if you are infected with these

**Comment by ImminentFate** [comment]
Curses, I should have checked for this before making my own detector: https://github.com/IridiumIO/Anti-Fractureiser

**Comment by ItzSwirlz** [comment]
> Curses, I should have checked for this before making my own detector: https://github.com/IridiumIO/Anti-Fractureiser

A UI frontend is still good though! However, idk if (.NET) will run on other systems

**Comment by ImminentFate** [comment]
> 

It won't sadly, (.NET will, but VB.NET certainly won't) but this is more for users who aren't technically inclined, most of whom will be on Windows anyway. The majority of people using linux will know how to use a shell script or use gradle to compile a jar file :)

**Comment by Krutonium** [comment]
> > 
> 
> It won't sadly, (.NET will, but VB.NET certainly won't) but this is more for users who aren't technically inclined, most of whom will be on Windows anyway. The majority of people using linux will know how to use a shell script or use gradle to compile a jar file :)

Not quite true - VB is just a language on top of .NET. What will prevent this from running on other systems is the use of XAML/UWP which is Windows only. If it was built using Windows Forms (And on dotnet instead of .NET (basically pre dotnet core days)) then it would work just fine on Mono. The same can be said about C# or any other language built on .Net. If it used Avalonia or ETO for the GUI Framework though, it work across all platforms.

**Comment by ImminentFate** [comment]
> > > 
> > 
> > 
> > It won't sadly, (.NET will, but VB.NET certainly won't) but this is more for users who aren't technically inclined, most of whom will be on Windows anyway. The majority of people using linux will know how to use a shell script or use gradle to compile a jar file :)
> 
> Not quite true - VB is just a language on top of .NET. What will prevent this from running on other systems is the use of XAML/UWP which is Windows only. If it was built using Windows Forms (And on dotnet instead of .NET (basically pre dotnet core days)) then it would work just fine on Mono. The same can be said about C# or any other language built on .Net. If it used Avalonia or ETO for the GUI Framework though, it work across all platforms.

I get what you're saying, but I laughed reading your answer because it basically boils down to "don't use the things you used" haha. 

.NET 6 is cross-platform, but you're right about the UI. I should've just used WinForms tbh, but WPF is nearly as easy and much nicer. 
Doesn't matter anyway, Curseforge will probably do a better of their tool in the end

**Comment by xyzeva** [comment]
This was implemented by [fc8b27b](https://github.com/fractureiser-investigation/fractureiser/commit/fc8b27b9ced4bd93570448e290021fff7482620f)

**Pull request closed without merging**